LICENCE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017 Crown Copyright (Government Digital Service)
```

Comment:
Noting, in case anyone wonders, that this date doesn't have to be the current year, so it's fine as-is.